src/components/Button/index.js

```diff
@@ -109,6 +109,9 @@ const propTypes = {
 
     /** Id to use for this button */
     nativeID: PropTypes.string,
+
+    /**  accessibility label for the component */
+    accessibilityLabel: PropTypes.string,
```

Comment:
This should be required

Reply:
does that mean all Buttons will need an accessibility label? & should adding all of them be in scope of this PR?

Reply:
Well since `accessibilityLabel` is required on `GenericPressable` it would make sense to have this required on `Button` too. I think it would be better to split this to two PRs:
1. Migrate from Pressable to PressableWithFeedback for Button
2. Disable PressableWithFeedback on press

cc @roryabraham thoughts on this?

Reply:
I agree that `accessibilityLabel` should be required. I also agree that we should start by disabling `PressableWithFeedback` on press in this PR, then create a separate PR to migrate from `Pressable` to `PressableWithFeedback` for `Button`.

Reply:
@roryabraham I have already done the migration part in this PR as we didn't get a reply for ~2 weeks, so what's the plan of action here?